eng/versioning/version_data.txt

```diff
@@ -4,7 +4,7 @@
 
 com.azure:azure-data-sdk-parent;1.3.0;1.3.0
 com.microsoft.azure:azure-batch;10.0.0;10.1.0-beta.1
-com.microsoft.azure:azure-applicationinsights-query;1.0.0-beta-2;1.0.0-beta-2
+com.microsoft.azure:azure-applicationinsights-query;1.0.0-Preview-1;1.0.0-Preview-2
```

Comment:
Using the same versioning scheme the first preview used to ensure users notice the updated version.
https://mvnrepository.com/artifact/com.microsoft.azure/azure-applicationinsights-query/1.0.0-Preview-1